lib/data/models/app/salawat.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:enum_to_string/enum_to_string.dart';
+import 'package:strings/strings.dart' as strings;
+
+enum Salah {
+  FAJR,
+  ZUHR,
+  ASR,
+  MAGHRIB,
+  ISHA,
+}
+
+extension SalahExt on Salah {
+  String getStringName() {
+    return strings.capitalize(EnumToString.convertToString(this).toLowerCase());
```

Comment:
why to lowercase then to capitalize?
and why not to use just this.toString().toUpperCase()

Reply:
capitalize on make the first character upper case